include/ode.hpp

```diff
@@ -591,6 +591,19 @@ constexpr auto drdt_chainreaction(double rp, double strength, double r, double m
     return result;
 }
 
+/**
+ * @brief Derivative of radius of solid fragment undergoing ablation in [m/s]
+ * 
+ * Source: Avramenko et al, 2014 [https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1002/2013JD021028]
+ * 
+ * @param r : meteoroid radius, in [m]
+ * @param m : meteoroid mass, in [kg]
+ * @param dmdt : ablation mass loss, in [kg/s]
+ */
+constexpr auto drdt_ablation(double r, double m, double dmdt) noexcept {
+    return r/m * dmdt/3;
```

Comment:
Re-use this function in `drdt_chainreaction()` -> then we don't have code duplication